src/openrct2/interface/Window.h

```diff
@@ -498,6 +499,20 @@ enum class VisibilityCache : uint8_t
     Covered
 };
 
+enum class CloseWindowModifier : uint8_t
+{
+    CLOSE_WINDOW_MODIFIER_NONE,
+    CLOSE_WINDOW_MODIFIER_SHIFT,
+    CLOSE_WINDOW_MODIFIER_CONTROL
+};
```

Comment:
With enum class, you do not need to prefix each item with the enum name. The casing should also be TitleCase.